spark/interpreter/src/main/java/org/apache/zeppelin/spark/Utils.java

```diff
@@ -149,7 +149,12 @@ static boolean isSpark2() {
   }
   
   public static String buildJobGroupId(InterpreterContext context) {
-    return "zeppelin-" + context.getNoteId() + "-" + context.getParagraphId();
+    String userIdAndHyphen = "";
+    if (context.getAuthenticationInfo() != null && context.getAuthenticationInfo().getUser() != null) {
```

Comment:
as in [here](https://github.com/apache/zeppelin/blob/master/spark/interpreter/src/main/java/org/apache/zeppelin/spark/Utils.java#L172) perhaps you should check if getUser() returns isEmpty()

Reply:
Oh, I'm sorry. I missed.
Thank you.